### PR TITLE
docs: Add Trivy migration guide

### PR DESCRIPTION
**Pull request by simar7**

### CODE REVIEW

File: docs/guides/trivy.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+```
+### With tfsec
+```shell
+$ tfsec <dir>
+```
+
+## Passing tfvars
+### With Trivy
+```shell
+$ trivy --tf-vars <vars.tf>
```
Comment (itaysk) [inline docs/guides/trivy.md:34]: missing target?

File: docs/guides/trivy.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Moving towards configuration scanning with Trivy
+Overtime we've taken [trivy][trivy] to be the go-to scanning tool for a vareity of things. This also includes terraform scanning.
+
+This section describes the differences between Trivy and tfsec.
+
+| Feature               | Trivy                                                  | tfsec                |
+|-----------------------|--------------------------------------------------------|----------------------|
+| Built-in Policies     | :material-check:                                       | :material-check:     |
+| Custom Policies       | Rego                                                   | Rego, JSON, and YAML |
+| Policy Metadata[^1]   | :material-check:                                       | :material-check:     |
```
Comment (itaysk) [inline docs/guides/trivy.md:10]: not sure what this was supposed to mean actually. I don't think this is any differentiator, wdyt?

File: docs/guides/trivy.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+| Show Successes        | :material-check:                                       | :material-check:     |
+| Disable Policies      | :material-check:                                       | :material-check:     |
```
Comment (itaysk) [inline docs/guides/trivy.md:12]: if we're maintaining tfsec style ignores it should be documented in trivy, if not, we should change this. anyway it probably deserves c clarification section here.

File: docs/guides/trivy.md (hunk 2 above)
Comment (simar7) [reply docs/guides/trivy.md]: yeah agree - I updated the table to only include the differences

File: docs/guides/trivy.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# Moving towards configuration scanning with Trivy
+Overtime we've taken [trivy][trivy] to be the go-to scanning tool for a vareity of things. This also includes terraform scanning.
+
+This section describes some differences between Trivy and tfsec.
+
+| Feature              | Trivy                                                  | tfsec                |
+|----------------------|--------------------------------------------------------|----------------------|
+| Policy Distribution | Embedded and Updated via Registry                      | Embedded             |
+| Custom Policies      | Rego                                                   | Rego, JSON, and YAML |
+| Supported Formats    | Dockerfile, JSON, YAML, Terraform, CloudFormation etc. | Terraform  Only      |
```
Comment (itaysk) [inline docs/guides/trivy.md:10]: wrong way?

Comment (simar7) [reply docs/guides/trivy.md:10]: wrong way in what sense? you want the tfsec column first?

**Comment by simar7** [comment]
@itaysk thanks for the review, I think you'll have to merge this PR as I don't seem to have admin / maintainer privileges in the tfsec repo. Or you could give me the right perms and I'll do it.

**Comment by AnaisUrlichs** [comment]
@itaysk @simar7 do you have to make a new release so this becomes visible in the docs?
Screenshot 2023-03-20 at 09 27 43

**Pull request merged**